subprojects/gradle/build-verdict-tasks-api/build.gradle.kts

```diff
@@ -0,0 +1,5 @@
+plugins {
```

Comment:
Create a separate module for the verdict API because I don't want modules depends on every plugin at the verdict plugin